dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -12,7 +12,14 @@ class MakerControllerTest < ActionController::TestCase
     @school = create :school
     @school_maker_high_needs = create :school, :is_maker_high_needs_school
 
-    @devices_2022 = ensure_script 'devices-2022'
+    # Create 3 versions of the devices unit where one is assigned to the user,
+    # one is the version the user most recently made progress in, and the other
+    # is the most recent version (for testing that MakerController.maker_script
+    # prioritizes an assigned unit version over a unit the user has progress in
+    # over the newest version).
```

Comment:
are you creating 3 versions or just 2? asking because I only see 2 `ensure_script` calls below

Reply:
Good point! I was creating 2 scripts and then using the actual latest-stable-version but I'll adjust it for consistency and clarity!